stock_picking_product_barcode_report/report/report_label_barcode_template.xml

```diff
@@ -18,7 +18,7 @@
                                     t-value="'02' + barcode + '10' + lot.name"
                                 />
                                 <img
-                                    t-att-src="'/report/barcode/?type=%s&amp;value=%s&amp;width=%s&amp;height=%s' % (type, value, 600, 100)"
+                                    t-att-src="'/report/barcode/?type=%s&amp;value=%s&amp;width=%s&amp;height=%s' % (type, value, 600, 60)"
```

Comment:
Why this change? Is doing it a bit shorter in height?

Reply:
This prevents a blank space from appearing below the barcode. What really determines the size is the element's style.

Reply:
Then I understand this should be backported to v15 as well.